packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -17,8 +17,14 @@ export const defaultAcceptanceMsg = `The offer has been accepted. Once the contr
 const getKeys = obj => harden(Object.getOwnPropertyNames(obj || {}));
 const getKeysSorted = obj =>
   harden(Object.getOwnPropertyNames(obj || {}).sort());
-
-export const makeZoeHelpers = zcf => {
+/**
+ * @function makeZoeHelpers - makes an object with helper functions useful to zoe contracts.
+ *
+ * @param {ContractFacet} zcf
+ */
+// zcf only picks up the type if the param is in parens, which eslint dislikes
```

Comment:
Huh, interesting. For what it is worth, prettier has moved to always using parens in 2.0.0 https://prettier.io/docs/en/options.html#arrow-function-parentheses